src/index.js

```diff
@@ -0,0 +1,219 @@
+"use strict";
+
+// import axios, { isCancel, AxiosError } from 'axios';
+
+const state = {
+    citySearchButton: null,
+    resetButton: null,
+    tempValue: 0,
+    lat: null,
+    lon: null,
+    upbutton: null,
+    downbutton: null
+};
+
+const changeCityHeader = () => {
+    const cityName = document.getElementById("city-spotlight");
+    const searchInput = document.getElementById("search-input").value;
+    cityName.innerHTML = `${searchInput}`;
+};
+
+const getLocation = () => {
+    let lat;
+    let lon;
+    return axios.get("http://127.0.0.1:5000/location", {
+        params: {
+            q: document.getElementById("search-input").value
+        }
+    })
+        .then((response) => {
+            state.lat = response.data[0].lat;
+            state.lon = response.data[0].lon;
+            getRtWeather();
+        })
+        .catch((error) => {
+            console.log("There's an error " + error);
+        });
+};
+
+const getRtWeather = () => {
+    axios.get("http://127.0.0.1:5000/weather",
+    {
+        params: {
+            lat: state.lat,
+            lon: state.lon,
+            units: "imperial"
+        }
+    })
+    .then((response) => {
+        state.tempValue = convertK2F(response.data.main.temp);
+        document.getElementById("temp-value").innerHTML = state.tempValue;
+        handleTempChanged();
+    })
+
+    .catch((error) => {
+        console.log("There's an error " + error);
+    })
+};
+
+const convertK2F = (temp) => {
+    return Math.trunc((temp - 273.15) * 9 / 5 + 32);
+}
+
+const handleSearchButtonClicked = () => {
+    changeCityHeader();
+    getLocation();
+};
+
+const handleResetBtnClicked = () => {
+    location.reload();
+}
+
+const handleUpBtnClicked = () => {
+    state.tempValue += 1;
+    console.log(state.tempValue);
```

Comment:
Remember to remove debugging log statements

Reply:
removed